src/components/icons/social/telegram-icon.jsx

```diff
@@ -6,6 +6,17 @@ let TelegramIcon = ({ className, height = '2.4rem' }) => (
     viewBox="0 0 240 240"
     className={className}
     height={height}>
+    <style>
```

Comment:
I start thinking that we did it in a wrong way: icon shouldn't be gray be default, can we move this logic outside? or at least add prop for controlling this

Reply:
@kitos sorry, but I don't understand what exactly do you want. You want control icon color by prop, right?

Reply:
yep, or by css